packages/core/src/__tests__/dispatcher-mock-ipfs.test.ts

```diff
@@ -225,10 +230,17 @@ describe('Dispatcher with mock ipfs', () => {
     )
     const queryID = queryMessageSent.id
 
-    // Handle UPDATE message
+    // Handle UPDATE message without model
     dispatcher.repository.stateManager.update = jest.fn()
     await dispatcher.handleMessage({ typ: MsgType.UPDATE, stream: FAKE_STREAM_ID, tip: FAKE_CID })
-    expect(dispatcher.repository.stateManager.update).toBeCalledWith(state$.id, FAKE_CID)
+
+    // Handle UPDATE message with model (to ensure optional use of model parameter)
+    dispatcher.repository.stateManager.update = jest.fn()
```

Comment:
the update function is already mocked on line 234, doing it again here is redundant.